auditlog/models/http_request.py

```diff
@@ -47,20 +45,14 @@ def current_http_request(self):
         first call.
         If no HTTP request is available, returns `False`.
         """
+
```

Comment:
Is this empty line required? It introduces a useless change which can create trouble when backporting patches between branches.